pkgs/tools/archivers/arc_unpacker/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, cmake, makeWrapper, boost, libpng, libjpeg, zlib
+, openssl, libwebp, catch }:
+
+stdenv.mkDerivation rec {
+  pname = "arc_unpacker";
+  version = "unstable-2019-01-28";
```

Comment:
Is there a reason to use this over the latest stable upstream release? If so, please open an upstream issue (politely) asking what they think about a new release. I think that's part of good open source manners, since it may happen that upstream gets bug reports from NixOS users that use an unstable version for some reason -- so there should be a good reason for that, and they should be aware of it.

If you decide to stick with the unstable version and open such an issue, please link it in a comment on the `version` attribute.

Reply:
The tool doesn't seem like it's being actively maintained. When reviewing, I assumed that a critical bugfix that's needed in the workflow of the PR author is present somewhere between latest release and HEAD of the main branch (the rev described in the PR points to the current HEAD at the time of writing this), and the commit is more than a year old.

Reply:
You're most likely right. Still: If that critical bugfix exists that knowledge should be made explicit. Even if upstream seems unmaintained, it can't hurt to give them notice. Maybe they just consider the current version "good enough" and an issue will prompt them to make that status "official" with a new release. If they ignore it, little is lost.

Reply:
The most direct reason for choosing the current `HEAD` was that latest release wouldn't build, though the fact that the latest release seems far behind `HEAD` did influence my choice.

Reply:
I've rebased and added a comment about the latest release not building.

Reply:
Issue opened upstream.